jberet-se/src/test/java/org/jberet/se/BatchSEEnvironmentTest.java

```diff
@@ -141,6 +142,21 @@ public void testCreateThreadPoolExecutor() throws Exception {
         }
     }
 
+    @Test
+    public void testResolveDbProps() {
+        BatchSEEnvironment batchSEObject = new BatchSEEnvironment();
+        Assert.assertEquals("foopass",
+                batchSEObject
+                        .getBatchConfigurationProperties()
+                        .get(org.jberet.repository.JdbcRepository.DB_PASSWORD_KEY)
+                        .toString());
+
+        Assert.assertEquals("foouser",
+                batchSEObject
+                        .getBatchConfigurationProperties()
+                        .get(JdbcRepository.DB_USER_KEY)
+                        .toString());
+    }
```

Comment:
You may want to add more unit tests just for testing `parseProp` method, without using `BatchSEEnvironment`, just to verify various input formats and expected output.

Reply:
tests added